src/React/DOM/Props.purs

```diff
@@ -297,6 +300,19 @@ radioGroup = unsafeMkProps "radioGroup"
 readOnly :: Boolean -> Props
 readOnly = unsafeMkProps "readOnly"
 
+ref :: String -> Props
+ref = unsafeMkProps "ref"
+
+-- | You can use `writeRef` to store a reference on `Refs`.
+-- | ``` purescrript
+-- | div [ withRef (writeRef this "inputElement") ] [...]
+-- | ```
+withRef
+  :: forall access eff
+   . (Ref -> Eff (refs :: ReactRefs (write :: Write | access) | eff) Unit)
```

Comment:
Looks great! Quick question did we want this to be `Nullable Ref`? Or are we thinking that the possibility the ref is null would be handled elsewhere.
https://facebook.github.io/react/docs/refs-and-the-dom.html#caveats

Reply:
Good point.  Having here `Nullable Ref` will be more explicit, so I prefer this. Then in the `React.DOM.refToNode` we will not need to use `unsafeCoerce`

Reply:
We will only need to add a comment, that if one is attaching the ref to `ReactThis` one should do that using the `Nullable Ref` rather than `Ref` itself.

Reply:
Great! Thanks!